spec/js/views/add_todo_view_spec.js

```diff
@@ -0,0 +1,49 @@
+define([
+  'views/add_todo_view',
+  'views/todos_view',
+  'models/todo',
+  'collections/todos',
+  'ejs'
+], function (AddToDoView, ToDosView, ToDo, ToDoCollection) {
+  var addToDoView;
+
+  beforeEach(function () {
+    var toDosView = new ToDosView({model: new ToDoCollection(), el: $('body')});
+
+    addToDoView = new AddToDoView({model: toDosView, el: $('body')});
```

Comment:
If you consider how Backbone#render in the view's `el` and how jasmine uses a single page to run tests, this will create issues. You must cleanup after your tests.